webapp/packages/core-blocks/src/Table/TableItemSelect.tsx

```diff
@@ -52,8 +52,8 @@ export const TableItemSelect: React.FC<Props> = observer(function TableItemSelec
   return styled(styles, checkboxStyles)(
     <Checkbox
       className={className}
-      disabled={context.selectDisabled || disabled}
-      checked={checked || context.isSelected()}
+      disabled={disabled ?? context.selectDisabled}
+      checked={checked ?? context.isSelected()}
```

Comment:
revert, the disabled property can't override the context, so logic is: if any of the two rules is true, then the checkbox is disabled or checked